refactor-content-spacing.md

```diff
@@ -0,0 +1,8 @@
+---
+"@commercetools-docs/ui-kit": minor
+"@commercetools-website/docs-smoke-test": patch
+---
+
+Markdown UI Kit: Refactor content spacing to be able to render markdown inside visual boxes using a new `TypographyContainer` wrapper.
+
+Markdown UI Kit: Adjust margins of lists
```

Comment:
This file should not exist twice - here and in `.changset` directory?

Reply:
Good catch, I think I did not pay close enough attention while rebasing my branch.  Fixed it.